src/java/org/apache/cassandra/repair/autorepair/AutoRepairConfig.java

```diff
@@ -59,7 +61,8 @@ public class AutoRepairConfig implements Serializable
     public enum RepairType implements Serializable
     {
         full,
-        incremental;
+        incremental,
+        preview_repaired;
```

Comment:
was playing around with adding preview repair and didn't realize I committed it when I did.  Not sure if I actually implemented it correctly, may back it out.